server/src/main/java/io/littlehorse/server/streamsimpl/coreprocessors/KafkaStreamsLHDAOImpl.java

```diff
@@ -68,18 +65,13 @@
 @Slf4j
 public class KafkaStreamsLHDAOImpl implements LHDAO {
 
-    private Map<String, NodeRun> nodeRunPuts;
     private Map<String, Variable> variablePuts;
     private Map<String, ExternalEvent> extEvtPuts;
```

Comment:
`ExternalEvent` was also hard to migrate because of the logic inside `getUnclaimedEvent()` method, so I also delayed this migration. I might need some help from someone who understands that logic :)

Reply:
Got it, yes, my branch has an implementation of that logic with the `GetableStorageManager` already, but it wasn't hooked up to the rest of the code. We can pair on that together.